components/brave_wallet/browser/test/BUILD.gn

```diff
@@ -43,6 +43,7 @@ source_set("brave_wallet_unit_tests") {
       "//brave/components/brave_wallet/browser",
       "//brave/components/brave_wallet/browser:ethereum_permission_utils",
       "//brave/components/brave_wallet/common",
+      "//brave/components/brave_wallet/common:common_constants",
       "//brave/components/brave_wallet/common:mojom",
       "//brave/components/brave_wallet/renderer/test:brave_wallet_response_unit_tests",
       "//chrome/browser",
```

Comment:
@darkdh This deps is not allowed in components, please move `keyring_controller_unittest.cc` to `browser/brave_wallet/test`

Reply:
https://github.com/brave/brave-browser/issues/17947